pkg/util/os/atexit/atexit.go

```diff
@@ -29,6 +30,13 @@ func Exit(code int) {
 	os.Exit(code)
 }
 
+// ExitWithMessage exit and print the error message.
```

Comment:
@hferentschik Hope this is what you wanted, I didn't put everything because let me clear about the requirement.